### ISSUE TITLE
Lack of focus ring or other visual focus indicators in "Top 3", "Top 5" or "All" dropdown in adoption insights

### ISSUE THREAD

**Issue by christoph-jerolimov** [redhat-developer/rhdh-plugins#3549]
## Description of problem:

When the user wants use the keyboard to change if the user wants see the top 3, 5, or "All" catalog entities in the adoptions insights plugin, the user doesn't see any visual indicator for the focus.

## Steps to Reproduce

1. In `workspaces/adoption-insights` run `yarn install`
2. In `workspaces/adoption-insights` run `yarn start` to start the Backstage app
3. Open the browser (use playwright-cli when possible) http://localhost:3000
4. Navigate to Adoption insights page
5. There is a card called "Top 3 catalog entities"
6. In the card footer there is a dropdown, select or menu that shows "Top 3", the user can select "Top 3", "Top 5" or "All".

<img width="256" height="158" alt="Image" src="https://github.com/user-attachments/assets/ff7521fe-f844-4649-ad51-c6cbb05c7512" />

## Actual results:

The user can change the value with a mouse click but not via keyboard.

In addition to the issue mentioned above (need to press tab in order to change the selection, while the menu is open), the toggle buttons in the select menu have their keyboard focus styles removed, making it hard to use via keyboard

## Expected results:

If the user tries to change the value via the keyboard it doesn't show an indicator.

Focus ring appears on focus. Ideally this menu should not be themed at all (the theme package should theme components, not plugins)
